public/app/features/dashboard-scene/scene/setDashboardPanelContext.ts

```diff
@@ -11,7 +11,16 @@ import { getDashboardSceneFor, getPanelIdForVizPanel, getQueryRunnerFor } from '
 import { DashboardScene } from './DashboardScene';
 
 export function setDashboardPanelContext(vizPanel: VizPanel, context: PanelContext) {
-  context.app = CoreApp.Dashboard;
+  const dashboard = getDashboardSceneFor(vizPanel);
+  context.app = dashboard.state.isEditing ? CoreApp.PanelEditor : CoreApp.Dashboard;
```

Comment:
this is comppleltly wrong :) did not catch it, just because your in dashboard edit mode does not mean your i panel editor :) 

Will fix in another PR

Reply:
oops :)